src/sid/plotting.py

```diff
@@ -171,11 +176,54 @@ def _create_y_ticks_and_labels(df):
     "ylabel": "Contact Model",
     "invert_yaxis": True,
     "colorbar": True,
+    "cmap": "YlOrBr",
 }
 
 
-def plot_infection_rates_by_contact_models(df_or_time_series, fig_kwargs=None):
-    """Plot infection rates by contact models."""
+def plot_infection_rates_by_contact_models(
+    df_or_time_series: Union[pd.DataFrame, dd.core.DataFrame],
+    show_reported_cases: bool = False,
+    numbers: str = "shares",
```

Comment:
I think `numbers` is not a very intuitive name. Maybe `units` or `metric` instead?

Reply:
agree. then ``unit`` and singular for the values like ``share``, ``population_share``, ``incidence``?

Reply:
Yes!